src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.jsx

```diff
@@ -70,12 +70,25 @@ LegacyConfigForm.propTypes = {
     allowAnonymousPosts: PropTypes.bool.isRequired,
     allowAnonymousPostsPeers: PropTypes.bool.isRequired,
     blackoutDates: PropTypes.string.isRequired,
-  }).isRequired,
+  }),
   onSubmit: PropTypes.func.isRequired,
   // eslint-disable-next-line react/forbid-prop-types
   formRef: PropTypes.object.isRequired,
   intl: intlShape.isRequired,
   title: PropTypes.string.isRequired,
```

Comment:
@davidjoy this title field is `isRequired` but in the tests `src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.test.jsx:defaultAppConfig` doesn't have title attribute. 

This causes the following warning. 
```
 PASS  src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.test.jsx (15.465 s)
  ● Console

    console.error
      Warning: Failed prop type: The prop `title` is marked as required in `LegacyConfigForm`, but its value is `undefined`.
          in LegacyConfigForm (created by ShimmedIntlComponent)
          in ShimmedIntlComponent (created by InjectIntl(ShimmedIntlComponent))
          in InjectIntl(ShimmedIntlComponent)
          in IntlProvider
```

Reply:
Ah!  I should fix that.  Will do - good catch.

Reply:
Fixed - the test should have included a `title` for the component.